legistar/bills.py

```diff
@@ -299,7 +300,14 @@ def attachments(self, matter_id):
     def votes(self, history_id):
         url = self.BASE_URL + '/eventitems/{0}/votes'.format(history_id)
 
-        response = self.get(url)
+        try:
+            response = self.get(url)
```

Comment:
scrapelib.scraper [raises 4xx and 5xx errors by default](https://github.com/jamesturk/scrapelib/blob/937cc99c08a326beca2025e4adc5ddd4dddc5b76/scrapelib/__init__.py#L200), so this seems like the right way to handle that. thanks!

that said, we also check for a 500 error in the `_missing_votes` method, fired in the next block.

https://github.com/opencivicdata/python-legistar-scraper/blob/f521c0d66a0a0ce35a9ccd9469abc8fa4e64dcce/legistar/bills.py#L437-L441

the way we have the scraper configured, i don't see how we'd ever get to that check, because an httperror would be raised in this block first. that seems like a bug – can it be patched in this pr?

Reply:
^ actual question. i'd be open to a patch in a separate pr, too, if you feel it's out of scope here.

Reply:
it's not a bug because, that 500 error won't raise an exception. I'll add a test to demonstrate this.

Reply:
i see, we've overridden `accept_response`. in that case, why handle the 404 here and not in the `_missing_votes` method?

Reply:
because generally, I do want `404`s to raise and error